app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  has_many :creators
+  belongs_to :user
```

Comment:
**[REQUIRED]**
Let's call the user that creates the event 'creator' 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))
We can implement it as follows
```
belongs_to :creator, class_name: "User"
```